### PR TITLE
Tan11389/style wms layers update

### PR DESCRIPTION
**Pull request by tanneryould**
Bug: Zoom out slowly until you see a "stretchy WMS layer"

Resolution:

* Update sample to set a MinScale of 7,000,000 - avoids zooming to polar regions
* Update sample to use SR 26915
* Update sample to use no basemap

### CODE REVIEW

**Comment by tanneryould** [comment]
Added a change to the title of the C++ sample. QML and other APIs ([QML](https://github.com/Esri/arcgis-runtime-samples-qt/tree/3dafd25d4d290cc94be475a635b2204e1eaf8808/ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer#style-wms-layer), [Java](https://github.com/Esri/arcgis-runtime-samples-java/tree/dev/ogc/style-wms-layer#style-wms-layer), [Android](https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/style-wms-layer#style-wms-layer)) refer to this sample as "Style WMS layer", but [C++](https://github.com/Esri/arcgis-runtime-samples-qt/tree/3dafd25d4d290cc94be475a635b2204e1eaf8808/ArcGISRuntimeSDKQt_CppSamples/Layers/StyleWmsLayer#style-wms-layers) and .[NET](https://github.com/Esri/arcgis-runtime-samples-dotnet/tree/a2b61971345bd3ea020b140fca01aa3d70f270e2/src/Android/Xamarin.Android/Samples/Layers/StyleWmsLayer#style-wms-layers) refer to it as "Style WMS layers".

With the difference between the two, we can't use the nifty [C++](https://developers.arcgis.com/qt/cpp/sample-code/style-wms-layers/)/[QML](https://developers.arcgis.com/qt/qml/sample-code/style-wms-layer/) code switcher in the samples docs. We should update one of them for consistency.

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml [hunk 1]
```diff
@@ -32,9 +32,7 @@ Rectangle {
         anchors.fill: parent
 
         Map {
-            Basemap {
-                initStyle: Enums.BasemapStyleArcGISImageryStandard
-            }
+            spatialReference: SpatialReference {wkid: 26915}
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml:36]:
```suggestion
minScale: 7000000

```

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml [hunk 2]
```diff
@@ -54,6 +52,8 @@ Rectangle {
                     // get the styles
                     styles = sublayer.sublayerInfo.styles;
                 }
+
+                minScale: 7000000
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml:56]:
```suggestion
```

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml [hunk 3]
```diff
@@ -32,9 +32,8 @@ Rectangle {
         anchors.fill: parent
 
         Map {
-            Basemap {
-                initStyle: Enums.BasemapStyleArcGISImageryStandard
-            }
+            spatialReference: SpatialReference {wkid: 26915}
+            minScale: 7000000
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/StyleWmsLayer/StyleWmsLayer.qml:36]:
```suggestion
            minScale: 7000000
            
```

**Review by ldanzinger — APPROVED** [review]

**Pull request merged**